src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/iOS/FormsCameraView.ios.cs

```diff
@@ -61,7 +61,13 @@ public FormsCameraView()
 			AddConstraints(NSLayoutConstraint.FromVisualFormat("H:|[mainView]|", NSLayoutFormatOptions.AlignAllTop, null, new NSDictionary("mainView", mainView)));
 		}
 
-		void SetStartOrientation()
+		public void SetOrientation()
```

Comment:
Why this is `public`?

Reply:
It is called from `CameraViewRenderer.ios.cs` to reset the Orientation values.

Reply:
If you turn the device and the orientation changes, this method is going to reset the camera orientation accordingly. Not just at the start.

Reply:
The original code is setting `previewLayerFrame`'s dimensions only, not the video orientation.

Reply:
how about changing it to `internal`?

Reply:
@pictos Done!

Reply:
@pictos there are few places in the same class that could be changed to `internal` as well, but I am thinking if the code cleanup ticket #1587 is approved, I will apply there. What do you think?